components/brave_rewards/browser/rewards_service.h

```diff
@@ -168,6 +168,9 @@ class RewardsService : public KeyedService {
   // TODO remove this hack when ads is moved to the same process as ledger
   virtual void SetCatalogIssuers(const std::string& json) = 0;
   virtual void AdSustained(const std::string& json) = 0;
+  virtual void GetAdsNotificationsHistory(
+      const uint64_t from_timestamp,
+      const uint64_t to_timestamp) = 0;
```

Comment:
`to_timestamp` should be renamed to `to_timestamp_seconds` as otherwise gets confusing with JS which is normally ms